packages/boot/src/mixins/boot.mixin.ts

```diff
@@ -53,6 +55,11 @@ export function BootMixin<T extends Constructor<any>>(superClass: T) {
       this.bind(BootBindings.BOOT_OPTIONS).toDynamicValue(
         () => this.bootOptions,
       );
+
+      this.bind(CoreBindings.APPLICATION_PACKAGE_JSON).toDynamicValue(() => {
+        const pkgFile = path.resolve(this.projectRoot, 'package.json');
+        return require(pkgFile);
```

Comment:
Should we do a try/catch here in case `package.json` path is resolved incorrectly or the file doesn't exist (not sure why that may happen ... but in case it does).